homebrew_challenge/src/components/RouteMapRules/RouteMapRules.css

```diff
@@ -0,0 +1,74 @@
+.route-map-rules {
+  color: #133a53;
+}
+
+.route-map-link {
+  text-decoration: none;
+  color: #133a53;
+}
+
+#lockdown.active,
+#phase-1.active,
+#phase-2.active,
+#phase-3.active,
+#phase-4.active {
+  background-color: #319bd5;
+  border: none;
+  border-radius: 0;
+}
+
+#lockdown,
```

Comment:
Still used?

Reply:
Sets toggle buttons group's styling - needed to set this prop because the corners on your toggle group were applied here also

Reply:
which toggle group styling was taking effect ? would be better to fix the previous toggle group (by making it more specific) rather than allow it to be set and then undo it.